src/NovelRT.Interop/Graphics/NrtTexture.cpp

```diff
@@ -20,7 +20,15 @@ extern "C" {
     }
 
     Texture* texturePtr = reinterpret_cast<Texture*>(targetTexture);
-    texturePtr->loadPngAsTexture(std::string(file));
+
+    try
+    {
+      texturePtr->loadPngAsTexture(std::string(file));
+    }
+    catch (const Exceptions::InvalidOperationException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?